.github/workflows/build-wsl.yaml

```diff
@@ -286,3 +288,29 @@ jobs:
           git add .
           git commit -m "Auto-update build info"
           git push origin master
+
+      - name: Checkout WSL repo
+        if: ${{ steps.modified-artifacts.outputs.needs-wiki-update == 'true' }}
+        run: |
+          mkdir -p ${{ env.codeDir }}
+          git clone https://github.com/ubuntu/wsl.git ${{ env.codeDir }} --depth 1
+      - name: Checkout ubuntu-archive-tools
+        if: ${{ steps.modified-artifacts.outputs.needs-wiki-update == 'true' }}
+        run: |
+          mkdir -p ${{ env.codeDir }}/uat
+          git clone https://git.launchpad.net/ubuntu-archive-tools ${{ env.codeDir }}/uat --depth 1
+
+      - name: Notify ISO Tracker
+        if: ${{ steps.modified-artifacts.outputs.needs-wiki-update == 'true' }}
+        env:
+          ISOTRACKER_USERNAME: ${{ secrets.ISOTrackerUsername }}
+          ISOTRACKER_PASSWORD: ${{ secrets.ISOTrackerPassword }}
+        run: |
+          [ -f /tmp/all-releases.csv ] || ${{ env.codeDir }}/wsl-builder/lp-distro-info > /tmp/all-releases.csv
+          # There might have been more than one build in the latest commit 
+          for build in $(git diff-tree --no-commit-id --name-only -r HEAD | grep buildid); do
+            WSLId=$(basename $build);
+            WSLId=${WSLId%-buildid.md};
+            buildid=$(cat $build)
+            PYTHONPATH=${{ env.codeDir }}/uat ${{ env.codeDir }}/notify-isotracker $WslID $buildid
```

Comment:
Can you move `notify-isotracker` under `wsl-buidler/`? This will make buid-related stuff outside of WSL.